nodejs/examples/rabbitmq.js

```diff
@@ -0,0 +1,30 @@
+const amqp = require('amqplib/callback_api');
```

Comment:
You are using the Promise-based API but this imports the callback one.
The Promise-based API is the default one so you can do `const amqp = require('amqplib');`